changelog.rst

```diff
@@ -0,0 +1,11 @@
+=========
+Changelog
+=========
+
+Changes in version 1.1.0
+------------------------
+- ``ExplainCollection`` now importable from top level like so: ``from pymongoexplain import ExplainCollection``
+
+Changes in version 1.0.0
+------------------------
+= Initial ``ExplainCollection`` API
```

Comment:
Should this be `=` or `-`?

Reply:
Oops, it should be == because "changelog" is == on both top and bottom.

Reply:
Prashant was talking about the `= Initial ``ExplainCollection`` API ` which needs to be `- Initial...`

Reply:
Done